README.md

```diff
@@ -104,16 +105,16 @@ the plugin:
 
 ### Develocity API
 
-API Manual: https://docs.gradle.com/enterprise/api-manual/
-API Documentation: https://docs.gradle.com/enterprise/api-manual/ref/2022.4.html
+- API Manual: https://docs.gradle.com/enterprise/api-manual/
+- API Documentation: https://docs.gradle.com/enterprise/api-manual/ref/2022.4.html
```

Comment:
I can't find the diff here 👓

Reply:
I prefixed each line with `-` to make them a bulleted list